src/client/rewards/CampaignFooter/CampaignButtons.js

```diff
@@ -174,19 +378,33 @@ export default class CampaignButtons extends React.Component {
   }
 
   render() {
-    const { intl, post, daysLeft } = this.props;
-
+    const {
+      intl,
+      numberOfComments,
+      daysLeft,
+      propositionStatus,
+      match,
+      user,
+      proposition,
+    } = this.props;
+    const isAssigned = get(proposition, ['objects', '0', 'assigned']);
+    const propositionUserName = get(proposition, ['users', '0', 'name']);
+    const reviewPermlink = get(proposition, ['users', '0', 'review_permlink']);
+    const propositionUserWeight = get(proposition, ['users', '0', 'wobjects_weight']);
     return (
       <div className="Buttons">
         <div className="Buttons__wrap">
           <div>
-            {`${intl.formatMessage({
-              id: 'campaign_buttons_reserved',
-              defaultMessage: 'Reserved',
-            })} - ${daysLeft} ${intl.formatMessage({
-              id: 'campaign_buttons_days_left',
-              defaultMessage: 'days left',
-            })}`}
+            {intl.formatMessage({
+              id: this.buttonsTitle.id,
+              defaultMessage: this.buttonsTitle.defaultMessage,
+            })}
+            {this.buttonsTitle.defaultMessage === 'Reserved'
+              ? `- ${daysLeft} ${intl.formatMessage({
+                  id: 'campaign_buttons_days_left',
+                  defaultMessage: 'days left',
```

Comment:
{this.buttonsTitle.defaultMessage === 'Reserved'
              && `- ${daysLeft} ${intl.formatMessage({
                  id: 'campaign_buttons_days_left',
                  defaultMessage: 'days left',
                })} `